wpimath/src/main/java/edu/wpi/first/math/geometry/Rotation2d.java

```diff
@@ -130,7 +131,7 @@ public Rotation2d(double x, double y) {
    *
    * @param angle The angle of the rotation.
    */
-  public Rotation2d(Measure<Angle> angle) {
+  public Rotation2d(Measure<AngleUnit> angle) {
     this(angle.in(Radians));
```

Comment:
Sorry if this is a silly question since I'm not very familiar with Java units (old or new), but why does this use `Measure<AngleUnit>` instead of `Angle`?

Reply:
For the widest compatibility. Math operations may return a generic `Measure` instance, and this lets those objects be used as inputs instead requiring the dimension-specific `Angle` type.

Reply:
Makes sense! Any particular reason why some other parameters don't do that? (e.g., Translation2d constructor)

Reply:
I think those are just inconsistent... that should be addressed